gnome-initial-setup/pages/summary/gis-summary-page.c

```diff
@@ -206,15 +231,7 @@ log_user_in (GisSummaryPage *page)
 
   if (gis_driver_is_in_demo_mode (GIS_PAGE (page)->driver))
     {
-      if (!gis_pkexec (LIBEXECDIR "/eos-demo-mode-force-gdm-restart",
-                       NULL,
-                       NULL,
-                       &error))
-        {
-          g_warning ("Failed to forcibly reload GDM settings, autologin may not work: %s",
-                     error->message);
-          g_clear_error (&error);
-        }
+      reload_systemd_service (g_application_get_dbus_connection (G_APPLICATION (driver)), "gdm.service");
```

Comment:
This is getting the session bus. The main instance of systemd, which you'd need to ask to tell `gdm.service` to reload, runs on the system bus.